cms/djangoapps/contentstore/views/course.py

```diff
@@ -434,8 +442,13 @@ def _accessible_courses_list_from_groups(request):
             except ItemNotFoundError:
                 # If a user has access to a course that doesn't exist, don't do anything with that course
                 pass
-            if course is not None and not isinstance(course, ErrorDescriptor):
-                # ignore deleted or errored courses
+
+            # Custom Courses for edX (CCX) is an edX feature for re-using course (aka master course) content.
+            # IF course is an instance of CCX and logged in user is the staff then do not show CCX(s) in his
+            # studio (aka cms) dashboard. Because for a single master course there can be many (no limit) CCX.
+            # Otherwise his dashboard can be fill with huge list of CCXs of multiple master course.
+            if course is not None and not isinstance(course, ErrorDescriptor) and not isinstance(course.id, CCXLocator):
```

Comment:
See edited comment above in https://github.com/edx/edx-platform/pull/10877/files#r49656974